pkg/trace/stats/aggregation.go

```diff
@@ -99,4 +148,12 @@ func NewAggregationFromGroup(g *pb.ClientGroupedStats) Aggregation {
 			Synthetics:  g.Synthetics,
 		},
 	}
+
+	if g.PeerTags != nil {
+		if !sort.StringsAreSorted(g.PeerTags) {
+			sort.Strings(g.PeerTags)
+		}
```

Comment:
Move this to inside peerTagsHash in case there are other callers in the future?

Reply:
Good idea!

Reply:
Although this may incur some more CPU cycles, it might be better to just make things more "programmer safe".